### DISCUSSION TITLE
Opción "Estoy viendo"

### DISCUSSION THREAD

**Discussion by magamess** [Paco8/plugin.video.movistarplus#30] [Q&A]
Buenas! Quizás no es una pregunta específica del Addon y es also de Kodi realmente, pero ¿hay opción de añadir una sección como en la app nativa de Movistar+ de "Estoy viendo", donde te salgan los programas/capítulos que están empezados y de esa manera acceder de forma rápida?

Gracias!

**Comment by Paco8** [comment]
Quizás sí se podría añadir.

**Comment by Paco8** [comment]
Acabo de añadir esa sección en la versión 0.8.8.
